OpenRA.Game/Graphics/SpriteFont.cs

```diff
@@ -49,22 +49,22 @@ public SpriteFont(string name, int size, SheetBuilder builder)
 
 		void PrecacheColor(Color c, string name)
 		{
-			using (new PerfTimer("PrecacheColor {0} {1}px {2}".F(name, size, c.Name)))
+			using (new PerfTimer("PrecacheColor {0} {1}px {2}".F(name, Size, c.Name)))
 				for (var n = (char)0x20; n < (char)0x7f; n++)
 					if (glyphs[Pair.New(n, c)] == null)
 						throw new InvalidOperationException();
 		}
 
-		public void DrawText(string text, float2 location, Color c)
+		public void DrawText(string text, float2 location, Color c, int lineSpacing = 0)
 		{
-			location.Y += size;	// baseline vs top
+			location.Y += Size;	// baseline vs top
 
 			var p = location;
 			foreach (var s in text)
 			{
 				if (s == '\n')
 				{
-					location.Y += size;
+					location.Y += Size + lineSpacing;
```

Comment:
It might be better to have the linespacing either inclusive of the height, or as a multiple of the default line spacing (like text editors etc do).